docs/_generate/item.html

```diff
@@ -49,4 +49,26 @@
 {% endhighlight %}
   </div>
   <% } %>
+  <% if (prop['sdk-support']) {%>
+  <table>
+    <caption>SDK Version Requirements</caption>
+    <thead>
+    <tr><th></th><th>Mapbox GL JS</th><th>Mapbox iOS SDK</th><th>Mapbox Android SDK</th></tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>Base support</td>
+      <td><%- prop['sdk-support'].basic.js ? '>= ' + prop['sdk-support'].basic.js : 'Not supported' %></td>
+      <td><%- prop['sdk-support'].basic.ios ? '>= ' + prop['sdk-support'].basic.ios : 'Not supported' %></td>
+      <td><%- prop['sdk-support'].basic.android ? '>= ' + prop['sdk-support'].basic.android : 'Not supported' %></td>
+    </tr>
+    <tr>
+      <td>Data-driven styling</td>
+      <td><%- (prop['sdk-support']['property-function'] || {}).js ? '>= ' + (prop['sdk-support']['property-function'] || {}).js : 'Not supported' %></td>
+      <td><%- (prop['sdk-support']['property-function'] || {}).ios ? '>= ' + (prop['sdk-support']['property-function'] || {}).ios : 'Not supported' %></td>
+      <td><%- (prop['sdk-support']['property-function'] || {}).android ? '>= ' + (prop['sdk-support']['property-function'] || {}).android : 'Not supported' %></td>
+    </tr>
```

Comment:
Did you consider allowing arbitrary fields rather than hardcoding `basic` and `property-function`? This would allow us to specify more nuanced information about specific properties as needed. 

For example, `icon-image` might want to specify 

{
  'sdk-support': {
    'Basic support': ...
    'Data-driven styling support`: ...
    'SDF support:' ...
  }
}